File: lua/1071.lua
local a = io.read("*n")
local b = io.read("*n")

local start, endVal, sum = a, b, 0

if a > b then
    start, endVal = b, a
end

for i = start + 1, endVal - 1 do
    if math.abs(i % 2) == 1 then
        sum = sum + i
    end
end

print(sum)

File: go/1071.go
package main

import (
	"fmt"
	"math"
)

func main() {
  var a, b, start, end, sum int
  fmt.Scanf("%d", &a)
  fmt.Scanf("%d", &b)

  start = a
  end = b
  
  if a > b {
    start = b
    end = a
  }

  for i := start + 1; i < end; i++ {
    if math.Abs(float64(i % 2)) == 1 {
      sum += i
    }
  }

  fmt.Println(sum)
}
